ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -41,21 +41,21 @@ ${UPGRADE_CONFIG_MAP}    upgrade-config-map
 *** Test Cases ***
 Verify PVC Size
     [Documentation]    Verify PVC Size after the upgrade
-    [Tags]      Upgrade
+    [Tags]      Upgrade    Dashboard
     Get Dashboard Config Data
     ${size}     Set Variable        ${payload[0]['spec']['notebookController']['pvcSize']}[:-2]
     Should Be Equal As Strings      '${size}'       '${S_SIZE}'
 
 Verify Pod Toleration
     [Documentation]    Verify Pod toleration after the upgrade
-    [Tags]      Upgrade
+    [Tags]      Upgrade    Dashboard
```

Comment:
This is not a UI test

Reply:
Since the toleration settings will change soon, we will probably have to remove this test.